casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -357,12 +358,10 @@ object BlockAPI {
       _.withCasper[ApiErr[LightBlockQueryResponse]](
         implicit casper =>
           for {
-            dag               <- casper.blockDag
```

Comment:
I know you didn't write this method name but we should probably call this function `findByDeploy`